schema/rdl_schema_0.1.json

```
@@ -1025,6 +1025,37 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Temporal": {
+			"title": "Temporal coverage",
+			"type": "object",
+			"description": "The temporal period that the dataset covers.",
+			"properties": {
+				"start": {
+					"title": "Start date",
+					"type": "string",
+					"description": "The date on which the period begins. This can refer to a year (YYYY), a month (YYYY-MM) or a day (YYYY-MM-DD) as appropriate for the dataset.",
```

Strictly speaking, months and years are not dates.

```suggestion
					"title": "Start",
					"type": "string",
					"description": "The start of the period. A date (YYYY-MM-DD) is recommended. However, where a date is not available, year-months (YYYY-MM) and years (YYYY) are supported.",
```